nautobot/circuits/templates/circuits/circuittermination_retrieve.html

```diff
@@ -60,17 +51,17 @@
                                 ({{ peer }})
                             {% endwith %}
                         {% else %}
-                            {% if perms.dcim.add_cable and object.site %}
+                            {% if perms.dcim.add_cable and object.location %}
```

Comment:
Is the `object.location` check here redundant with the check at line 31/28 above?